src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/ManagedIndexUtils.kt

```diff
@@ -402,3 +406,9 @@ fun ManagedIndexConfig.shouldChangePolicy(managedIndexMetaData: ManagedIndexMeta
 
     return true
 }
+
+val ManagedIndexMetaData.wasReadOnly: Boolean?
```

Comment:
Can do:
`return this.actionMetaData?.actionProperties?.getBoolean(ActionProperties.WAS_READ_ONLY) == true`
To convert the response to just a `Boolean` instead of `Boolean?`

Reply:
Good suggestion, changed.